website/addons/s3/requirements.txt

```diff
@@ -1 +1 @@
-boto==2.31.1
```

Comment:
Took me a bit to recall the exact issue here, it was related to the osfstorage file audit scripts which run weekly. They required boto to perform uploads when synchronizing w/ Amazon Glacier.

1. We could test with Boto3 and if this is resolved we could just update requirements. (best solution)
2. We could make a separate virtual environment for the scripts, maybe a `requirements/scripts.txt`.

[boto 2.31.1 to fix utf encoding issues with multipart uploads](https://github.com/CenterForOpenScience/osf.io/commit/f048c91ea3c658d0e8ab1338f99c9a328771aff2)